minetest.conf.example

```diff
@@ -1961,16 +1961,6 @@
 #    type: int
 # main_menu_mod_mgr = 1
 
-#    type: string
-# modstore_download_url = https://forum.minetest.net/media/
-
-#    type: string
-# modstore_listmods_url = https://forum.minetest.net/mmdb/mods/
-
-#    type: string
-# modstore_details_url = https://forum.minetest.net/mmdb/mod/*/
-
 #    Print the engine's profiling data in regular intervals (in seconds). 0 = disable. Useful for developers.
 #    type: int
 # profiler_print_interval = 0
-
```

Comment:
I dunno, I didn't do it - Atom did. Maybe there were two lines at the end? Atom always ensures that there is specifically *one* newline at the end of the file.